pkg/component/vpa/updater.go

```diff
@@ -159,6 +159,8 @@ func (v *vpa) reconcileUpdaterDeployment(deployment *appsv1.Deployment, serviceA
 						fmt.Sprintf("--updater-interval=%s", durationDeref(v.values.Updater.Interval, gardencorev1beta1.DefaultUpdaterInterval).Duration),
 						"--stderrthreshold=info",
 						"--v=2",
+						"--kube-api-qps=100",
```

Comment:
I wonder if existing clusters which do not have an  --eviction-rate-limit configured, could be negatively affected by QPS that high. Looking at updater's eviction loop, I'm not quite certain that it runs at 1 query/eviction, but it seems close to that.

Since these are evictions, the QPS load on the cluster would be greatly amplified. I suspect that at 100 evictions/second the Kapi may become a bottleneck.

Reply:
Yeah, this definitely increases the load on the kube-apiserver! When we did this increase for e.g. the vpa-recommender, we didn't see a big impact on the apiserver side, so I'm kind of optimistic that this doesn't cause load that it cannot deal with.